EIPS/eip-draft_request_id.md

```diff
@@ -0,0 +1,155 @@
+---
+eip: <to be assigned>
+title: "eth/66: request identifier"
+author: Christoph Burgdorf <christoph@ethereum.org>
+discussions-to: https://github.com/ethereum/EIPs/issues/2482
+status: Draft
+type: Standards Track
+category: Networking
+created: 2020-01-17
+requires: 2464
+---
+
+<!--You can leave these HTML comments in your merged EIP and delete the visible duplicate text guides, they will not appear and may be helpful to refer to if you edit it again. This is the suggested template for new EIPs. Note that an EIP number will be assigned by an editor. When opening a pull request to submit your EIP, please use an abbreviated title in the filename, `eip-draft_title_abbrev.md`. The title should be 44 characters or less.-->
+
+## Simple Summary
+<!--"If you can't explain it simply, you don't understand it well enough." Provide a simplified and layman-accessible explanation of the EIP.-->
+This document proposes a way to slightly increase the efficiency of the `eth` networking protocol while at the same time reducing complexity in Ethereum node implementations.
```

Comment:
This is just personal preference so definitely feel free to ignore me, but I don't think it's a slight improvement! This could let nodes sync much faster; I think "This document proposes a way to increase the efficiency [...]" would be more accurate and more compelling.